tools/sgen/sgen-grep-binprot.c

```diff
@@ -4,7 +4,17 @@
 #include <glib.h>
 #include <unistd.h>
 #include <fcntl.h>
-
+#include <config.h>
+
+#if SIZEOF_VOID_P == 4
+typedef int		mword;
```

Comment:
If we're using exact types, then we should use the proper exact C types.  See `stdint.h`.